perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -729,6 +731,16 @@ sub setup_categories_and_bodies : Private {
             } else {
                 $category_extras_hidden{$contact->category} = $all_hidden;
             }
+
+            my $all_notices = (grep {
+                ( $_->{variable} || '' ) ne 'false'
+                && !$c->cobrand->category_extra_hidden($_)
+            } @$metas) ? 0 : 1;
+            if (exists($category_extras_notices{$contact->category})) {
+                $category_extras_notices{$contact->category} &&= $all_notices;
+            } else {
```

Comment:
I'm not sure I understand how you would end up with this already existing?

Reply:
(Meant to reply earlier, oops) I think it's possible in the situation where two overlapping bodies have the same category name, e.g. district/county or TfL/London boroughs.